pkg/mount/block/blockdev.go

```diff
@@ -173,25 +160,19 @@ func GetBlockStats() ([]BlockDev, error) {
 		devnames = append(devnames, rel)
 		return nil
 	})
-	if err != nil {
-		return nil, err
-	}
+
+	var blockdevs []*BlockDev
 	for _, devname := range devnames {
-		buf, err := ioutil.ReadFile(fmt.Sprintf("%s/%s/stat", root, devname))
-		if err != nil {
-			return nil, err
-		}
-		bstat, err := BlockStatFromBytes(buf)
-		if err != nil {
-			return nil, err
-		}
-		devpath := path.Join("/dev/", devname)
+		devpath := filepath.Join("/dev/", devname)
```

Comment:
Could the body of this for-loop be inside the `Walk`? Then you don't need the for-loop.

Reply:
that's just plain wrong, moved